### DISCUSSION TITLE
Kind of "Energy Dashboard" graphs ...

### DISCUSSION THREAD

**Discussion by cedricdelecole** [flixlix/energy-period-selector-plus#12] [Q&A]
Hello, I am using the "Power flow card plus" and "Energy flow card plus" in my dashboard, but was looking for an historical graphical representation like the Standard Energy dashboard does like this (Day / Week / Month / ...). Before installing the "Energy-period-selector-plus", I would like to know if this is the one which does the job, or if there was something else ?
I was looking for something like this :

![image](https://github.com/flixlix/energy-period-selector-plus/assets/135960043/b6d3aac5-05fd-4d17-bb11-f3f84ff39515)

Many thanks !!

**Comment by cedricdelecole** [comment]
P.S. - I have installed the card, but no data on graph is showned as in the description of the card :

![image](https://github.com/flixlix/energy-period-selector-plus/assets/135960043/59543c74-24de-47a3-82cd-e9d8369bdf7e)


here is what I have :

![image](https://github.com/flixlix/energy-period-selector-plus/assets/135960043/2a54a43b-7e02-4728-bc07-f18a84561ba7)


I think I am missing something
;-)

**Comment by flixlix** [comment]
I think this is what you are looking for, try using adding this card to your Lovelace:

```yaml
type: energy-usage-graph
```

**Reply by cedricdelecole** [reply]
Thank you vlixlix, but I can not find it ... nor in your packages, nor in HACS ... can you please link it to me ?
Many thanks !

**Reply by flixlix** [reply]
These energy cards I am mentioning are built in to HA. You just need to use this config in the yaml editor
https://www.home-assistant.io/dashboards/energy/
